core/cmd/hoverfly/main.go

```diff
@@ -283,6 +282,8 @@ func main() {
 		requestCache = nil
 	}
 
+	cfg.DisableProxyAuthoriation = *disableProxyAuth
```

Comment:
That is hoverctl. This flag is not implemented in hoverctl. The reason for this that I wanted to quickly implement it intime for the Hoverfly Cloud private beta. 

I could change the `-disable-standard-proxy-auth` flag on Hoverfly to be something like `-proxy-auth-type`.